apps/dashboard/src/lib/utils/translations/de.json

```
@@ -862,6 +862,9 @@
           "description_live": "Gesperrte Lektionen und Übungen bleiben den Schülern verborgen, bis Sie sie entsperren – nützlich, wenn Sie Inhalte Woche für Woche veröffentlichen.",
           "unlock_all": "Schalte alle frei"
         },
+        "certificate_deadline_required": "Legen Sie zuerst die Frist für das Zertifikat fest",
```

Biome (2.5.7)

[error] The key certificate_deadline_required was already declared.

(lint/suspicious/noDuplicateObjectKeys)